packages/ui/src/components/primitives/message/Embed.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { MessageProps } from './props';
+import { cn } from 'packages/ui/src/utils/utils';
+import { BlueLink } from '../ui/blue-link';
+
+const EmbedText = (props: {
+	children: React.ReactNode;
+	className?: string;
+}) => (
+	<span className={cn(`font-light text-primary`, props.className)}>
+		{props.children}
```

Comment:
How does this handle discord embeds? I think if you put in our existing embed parsing logic it should work here

Reply:
Fixed!